app/assets/stylesheets/skills_page.scss

```diff
@@ -0,0 +1,32 @@
+$base        : #AAAAAA;
+$secondary-color: white;
+
+.side-bar {
+  background-color: $secondary-color;
+  border: 25px $secondary-color solid;
+  border-radius: 6px;
+  margin-top: 20px;
+  padding: 0;
+}
+
+#fl-img-sidebar {
+border-bottom-color: black;
+border-bottom-style: dotted;
+}
+
+#panel2-1, #panel2-2 {
+  padding: 0;
+}
+
+.skill-select {
+font-family: monospace;
+}
+
+li.skill-item {
+  padding-bottom: 0px;
```

Comment:
Properties should be ordered margin-bottom, padding-bottom<br>`0px` should be written without units as `0`